view/components/utils/totals-in-matrix.js

```diff
@@ -8,6 +8,7 @@ var generateTableBody = function (rows, master) {
 	return rows.map(function (row) {
 		return tr(th(row.label, span({ class: 'hint' }, row.inputHint)),
 			list(row.paths, function (path) {
+				if (!path) return td();
```

Comment:
Let's rely on agreed explicit `null` comparison here (sloppy one):

`if (path == null) return td();`

We should treat it as contract, other falsy values like `''` or `false`, should not yield this result and should be treated as accidental errors.